cmd/query/app/token_propagation_handler.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"net/http"
+	"strings"
+
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+)
+
+func bearTokenPropagationHandler(logger *zap.Logger, h http.Handler) http.Handler {
```

Comment:
As much as a I prefer your version, the function name should be `bearer` instead of `bear`: `bearerTokenPropagationHandler`

Reply:
Ups, a typo I'll rename! Thanks